saleor/checkout/models.py

```diff
@@ -191,6 +191,9 @@ class Checkout(models.Model):
     )
 
     price_expiration = models.DateTimeField(default=timezone.now)
+    # Expiration time of the applied discounts.
+    # Decides if the discounts are updated before tax recalculation.
+    discount_price_expiration = models.DateTimeField(default=timezone.now)
```

Comment:
I would suggest to rename it to something without `_price_`  keyword. I mean, this flag is not used to refresh the prices. It is used to build the discount objects that will be used to refresh the prices, right?

Reply:
Hm. It basically updated `checkout.discount_amount` but I can change to `discount_expiration`